components/auto_segmentation/install.sh

```diff
@@ -3,11 +3,11 @@
 
 set -e
 
-MASK_RCNN_URL=https://github.com/matterport/Mask_RCNN
+MASK_RCNN_URL=https://github.com/onepanelio/Mask_RCNN
```

Comment:
The same as above.

Reply:
We made several changes in Mask_RCNN repo to make it work with multiple gpus. So, if we change this then it won't work with multiple GPUs.